src/shogun/base/SGObject.cpp

```diff
@@ -344,12 +339,7 @@ bool CSGObject::parameter_hash_changed()
 {
 	SG_DEBUG("entering\n")
 
-	uint32_t hash=0;
-	uint32_t carry=0;
-	uint32_t length=0;
-
-	get_parameter_incremental_hash(hash, carry, length);
-	hash=CHash::FinalizeIncrementalMurmurHash3(hash, carry, length);
+	auto hash = std::hash<CSGObject>{}(*this);
```

Comment:
Could this method now be const?

Reply:
yeah as soon as the problem with clang is solved...